packages/ui/src/components/Visualization/Canvas/CanvasForm.tsx

```diff
@@ -31,7 +31,7 @@ export const CanvasForm: FunctionComponent<CanvasFormProps> = (props) => {
     setSchema(schemaServiceRef.current.getSchemaBridge(visualComponentSchema?.schema));
 
     // Overriding parameters with an empty object When the parameters property is mistakenly set to null
-    if (visualComponentSchema?.definition.parameters === null) {
+    if (visualComponentSchema?.definition?.parameters === null) {
```

Comment:
In the failing test case, the `definition` property was `null`, hence parameters cannot be accessed. The fix is to add an optional chaining to ensure that we have a value.